app/src/main/java/org/sopt/alami/presentation/main/MainScreen.kt

```diff
@@ -0,0 +1,41 @@
+package org.sopt.alami.presentation.main
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.navigationBarsPadding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import kotlinx.collections.immutable.toPersistentList
+import org.sopt.alami.core.designsystem.theme.AlarmiTheme
+
+@Composable
+fun MainScreen(
+    navigator: MainNavigator = rememberMainNavigator()
+) {
+    Scaffold(
+        modifier = Modifier
+            .fillMaxSize(),
+
+        content = { padding ->
+            MainNavHost(
+                navigator = navigator,
+                padding = padding,
+                modifier = Modifier
+            )
+        },
+        bottomBar = {
+            MainBottomBar(
+                modifier = Modifier
+                    .background(color = AlarmiTheme.colors.grey800)
+                    .navigationBarsPadding(),
+                isVisible = navigator.showBottomNavigator(),
+                tabs = MainTabType.entries.toPersistentList(),
+                currentTab = navigator.currentTab,
+                onTabSelected = { navigator.navigate(it) }
```

Comment:
```suggestion
                onTabSelected = navigator::navigate
```

p1
위처럼 작성이 가능한데 차이점을 아시나요? 제가 꿀팀공유에 써놨답니다~

Reply:
`navigator::navigate`는 성능최적화를 고려한 메소드 레퍼런스 방식입니다 
함수를 직접 참조하여 추가 람다 객체를 생성하지 않는다는 장점이 있네요..!! 
감사합니다 👍